index.js

```diff
@@ -0,0 +1,23 @@
+const express = require('express');
+const cors = require('cors');
+const mongoose = require('mongoose');
+
+const app = express();
+
+app.use(cors());
+app.use(express.json());
+
+const uri =
+  'mongodb+srv://Cesar-Diaz:vacivusVoid1%40@cluster0.ybria.mongodb.net/Agora?retryWrites=true&w=majority';
```

Comment:
Try to use .env file and store passwords and links to access to an external server.

Reply:
I just pushed the server.js (renamed index.js) with the reference to the .env file and installed dotenv.